docs/contact.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
It looks like you made your lab2 branch at a point before adding your lab1 code.  I'm seeing both your lab1 and lab2 changes here.

Reply:
Other than that, your code looks good.